RULES.md

```diff
@@ -40,3 +39,40 @@ FROM {{ source('source_name', 'table_name') }}
 SELECT *
 FROM {{ ref('script_name') }}
 ```
+
+
+# No references outside of its staging area
+There are staging model(s) referencing a staging model that belongs in a different area
+
+If you want to cross domains, this should be done at mart level.
```

Comment:
There is mixed use of "area" and "domain". I think we should consistently use one or the other

Reply:
i'll go for area